Automattic/jetpack
# Contact Form: Update Form block placeholder styles

#### Changes proposed in this Pull Request:

* Improves the Form block placeholder styles
  *  Reduces the variation buttons size to fit the items better when all variations are available, and the theme has a smaller block container width
  * Changes the footer links alignment to the right

Before 
![image](https://user-images.githubusercontent.com/7811225/206703602-8f9b1e12-e9d4-4559-aa2e-6534c74eb3b5.png)

After
![image](https://user-images.githubusercontent.com/7811225/206704011-9372c359-ac33-40ec-8d1d-d99f0371b343.png)

 
#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a post and add a Form block
* Check if the Form block placeholder matches the changes proposed above
